packages/cli/test/unit/commands/login.test.ts

```diff
@@ -87,9 +87,9 @@ describe('login', () => {
         `Success! Email authentication complete for ${user.email}`
       );
 
-      await expect(client.stderr).not.toOutput(emoji('tip'));
-
       await expect(exitCodePromise).resolves.toEqual(0);
+
+      await expect(client.getFullOutput()).not.toContain(emoji('tip'));
```

Comment:
When we do `.not.toOutput` it seems like that would guarantee that our test will wait the full 3 seconds for each one of those statements. But i wonder if we can instead just assert that the "full output" doesn't contain the string, as shown here